Airports/EGPK/Freetext.txt

```diff
@@ -0,0 +1,16 @@
+;EGPK SRA Advisory Altitudes
+N055.33.31.010:W004.44.15.626:EGPK SRA 12 altitudes:5-1600'
+N055.33.00.217:W004.42.42.973:EGPK SRA 12 altitudes:4-1290'
+N055.32.29.768:W004.41.16.613:EGPK SRA 12 altitudes:3-980'
+N055.31.56.898:W004.39.41.042:EGPK SRA 12 altitudes:2-660'
+N055.30.54.962:W004.36.40.262:EGPK SRA 12 altitudes:THR-38'
+N055.34.47.405:W004.30.59.963:EGPK SRA 21 altitudes:5-2000'
+N055.33.54.466:W004.31.41.416:EGPK SRA 21 altitudes:4-1610'
+N055.32.58.963:W004.32.24.061:EGPK SRA 21 altitudes:3-1230'
+N055.32.03.966:W004.33.03.042:EGPK SRA 21 altitudes:2-840'
+N055.30.12.269:W004.34.32.893:EGPK SRA 21 altitudes:THR-60'
+N055.27.33.238:W004.26.59.348:EGPK SRA 30 altitudes:5-2000'
+N055.28.04.033:W004.28.25.708:EGPK SRA 30 altitudes:4-1610'
+N055.28.36.904:W004.30.03.582:EGPK SRA 30 altitudes:3-1230'
+N055.29.06.315:W004.31.27.063:EGPK SRA 30 altitudes:2-840'
+N055.30.08.726:W004.34.37.609:EGPK SRA 30 altitudes:THR-65'
```

Comment:
```suggestion
N055.30.08.726:W004.34.37.609:EGPK SRA 30 altitudes:THR-60'
```
60 according to [this](https://www.aurora.nats.co.uk/htmlAIP/Publications/2021-04-22-AIRAC/graphics/155421.pdf)

Reply:
Ive updated the above. The THR elevation for RWY 30 I believe is 65 ft [https://www.aurora.nats.co.uk/htmlAIP/Publications/2021-04-22-AIRAC/graphics/200797.pdfl](url)

Cheers

Reply:
nevermind, I'm blind